back/engines/free/email_campaigns/app/mailers/email_campaigns/application_mailer.rb

```diff
@@ -25,7 +27,7 @@ def campaign_mail
     private
 
     def show_unsubscribe_link?
-      true
+      user && campaign.class.try(:consentable_for?, user)
```

Comment:
I'm not 100% sure about it, but probably we don't need to call `consentable_for?` here.  `show_unsubscribe_link?` seems to be rather a property of the campaign class than a property of the user-campaign relationship.

Would it be enough to check it like this?

```suggestion
      user && campaign.class.respond_to?(:consentable_for?)
```

It's a bit easier to read, although the result would be the same in all normal cases (not bugs).